### PR TITLE
go-centrifuge Slides

### PR DESCRIPTION
**Pull request by vimukthi-git**
<This is intended as a guide when describing-reviewing pull requests for go-centrifuge>
Issue Link: <Github link>

### CODE REVIEW

**Comment by vedhavyas** [comment]
Let's move this to a new repo called presentations. We can have a single repo for all the talks/presentations done so far and any in the future there.

CC: @lucasvo

**Comment by vimukthi-git** [comment]
@vedhavyas there are somethings we need to discuss before doing that, part of it I was going to show in the presentation it self.
I don't know if you read the presentation but part of the reason why this here is explained in the slides them selves. Also slides refer to code in the repo, so they won't go out of sync as a reference.

File: presentation/presentation.slide [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+* Wait but first about Errors :)
+
+We have a few conventions around error handling,
+- All errors should be checked and propagated up the call chain (no panic or fatal)
+- Use constant error values
+- Use *github.com/centrifuge/go-centrifuge/errors* package to create errors
+
+* Bootstrappers
+
+.code ../bootstrap/bootstrapper.go /Bootstrapper/,/^}/
```
Comment (vimukthi-git) [inline presentation/presentation.slide]: @vedhavyas this is an example of a code reference

File: presentation/presentation.slide [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+- A model must not have any runtime dependencies such as database. Using a compile time dep such as a lib is fine
+- A model must wrap all coredocument operations
+- No direct access to internal fields must be allowed
+
+* Document Services
+
+- A Service must be defined to handle all business logic related to a single Model.
+- Some service functions are generic enough for them to be in the `DefaultService`.
+
+* Service Interface
```
Comment (charleenfei) [inline presentation/presentation.slide]: Validators can be added as a slide to this presentation :)

Comment (manuelpolzhofer) [reply presentation/presentation.slide]: my additional slide ideas:
- ethereum testnet/ centrifuge testnet
- multi-account
- command line interface

File: presentation/presentation.slide (hunk 1 above)
Comment (vedhavyas) [reply presentation/presentation.slide]: You can always refer this from another repo AFAIK

Comment (vimukthi-git) [reply presentation/presentation.slide]: I checked, it wouldn't include the code blocks with another repo link. Anyway these slides are intended as documentation, Its not a presentation per se. So I think we can just keep it in the repo it self. Its also easier to keep it update with code changes this way.

File: presentation/presentation.slide (hunk 2 above)
Comment (vimukthi-git) [reply presentation/presentation.slide]: @manuelpolzhofer I believe what you mean by `command line interface` is how it works technically? I will make a note of that.
Regarding `multi-account` there is already a slide here `Accounts` for that, what would you like it to be covered?

**Pull request closed without merging**